### PR TITLE
fix(TS): some left overs

### PR DESCRIPTION
**Pull request by ShaMan123**
## Motivation

left overs for recent merges

### CODE REVIEW

File: src/Pattern/index.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
 export { Pattern } from './Pattern';
-export type * from './types';
+export * from './types';
```
Comment (ShaMan123) [inline src/Pattern/index.ts:2]: TS complains.
I see the bundle grew so this might not be good

Comment (ShaMan123) [reply src/Pattern/index.ts:2]: and rather annoying

Comment (ShaMan123) [reply src/Pattern/index.ts:2]: I have no idea why the bundle grows

**Review by asturur — APPROVED** [review]

**Comment by asturur** [comment]
Probably every ts warning apart sourcemaps should be considering a blocking error

**Comment by ShaMan123** [comment]
> Member

**Comment by ShaMan123** [comment]
> Probably every ts warning apart sourcemaps should be considering a blocking error

agreed, why not source maps?

**Comment by ShaMan123** [comment]
We just need to throw when a ts warning is fired
https://github.com/fabricjs/fabric.js/blob/d7bb027058fc0ce30dbc2b610336a7bcc4756e7a/rollup.config.mjs#L35

**Pull request merged**